.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: ruby
+rvm:
+  - 2.5.1
+
+env:
+  global:
+    - RUBY_GC_MALLOC_LIMIT=90000000
+    - RUBY_GC_HEAP_FREE_SLOTS=200000
+    - SECURITY_SECRET_TOKEN=traviscitraviscitraviscitraviscitravisci
+
+services:
+  - postgresql
+  - redis-server
+
+before_install:
+  #- "echo '--colour' > ~/.rspec"
+  - "echo 'gem: --no-document' > ~/.gemrc"
+  # Solves rainbow issue: https://github.com/sickill/rainbow/issues/48
+  - gem update --system
+
+before_script:
+  - cp config/database.travis.yml config/database.yml
```

Comment:
I think we could just remove this line and the `config/database.travis.yml` file. It should work with just the `database.yml`. Let's try it out